docs/source/build_network.rst

```diff
@@ -1,1466 +1,16 @@
 Building Your First Network
 ===========================
 
-.. note:: The Build your first network (BYFN) tutorial has been deprecated. If
-          you are getting started with Hyperledger Fabric and would like to deploy
+.. note:: The Build your first network (BYFN) tutorial has been removed.
+          If you are getting started with Hyperledger Fabric and would like to deploy
           a basic network, see :doc:`test_network`. If you are deploying Fabric
           in production, see the guide for :doc:`deployment_guide_overview`.
 
-The build your first network (BYFN) scenario provisions a sample Hyperledger
-Fabric network consisting of two organizations, each maintaining two peer
-nodes. It also will deploy a Raft ordering service by default.
+          You can still find the BYFN tutorial in earlier Fabric documentation versions:
```

Comment:
Sorry Chris, the content of a `note` block needs to be left aligned with the `note` so all of the lines below the line with `note` on it need to be moved to the left to align top to bottom with `note`, right now the content is tabbed:

![image](https://user-images.githubusercontent.com/9400927/85895399-a8b30900-b7c4-11ea-963d-d9abf1db2210.png)

Reply:
Outdated, looks good now